ztag/transforms/mssql.py

```diff
@@ -22,12 +22,15 @@ def _transform_object(self, obj):
         if not results:
             return zout
 
-        # Version is required, ignore this record if it isn't present.
-        zout.transformed["version"] = results["version"]
+        to_copy = ["version", "instance_name", "encrypt_mode"]
 
-        if "instance_name" in results:
-            zout.transformed["instance_name"] = results["instance_name"]
-        if "prelogin_options" in results:
-            self.load_prelogin_options(results["prelogin_options"], zout)
+        for f in to_copy:
+            if results.get(f) is not None:
+                zout.transformed[f] = results[f]
+
+        # If we don't have a root encrypt_mode, fall back to prelogin_options
+        if zout.transformed.get("encrypt_mode") is not None:
```

Comment:
> If we don't have a root encrypt_mode, fall back to prelogin_options

Shouldn't this line then read `if zout.transformed.get("encrypt_mode") is None:` , where currently it checks `is not None`?

Reply:
Yes, it should...good eye.